services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: str
+) -> QuerySet[MovieSession]:
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    movie = Movie.objects.get(id=movie_id)

```

Comment:
The `create_movie_session` function uses `get()` to retrieve `CinemaHall` and `Movie` by ID. If the IDs do not exist, this will raise a `DoesNotExist` exception. Consider adding exception handling to manage this scenario gracefully.